api-spec/protobuf/ark/v1/service.proto

```diff
@@ -91,6 +102,7 @@ message CompletePaymentResponse {}
 
 message RegisterPaymentRequest {
   repeated Input inputs = 1;
+  optional string ephemeral_pubkey = 2;
```

Comment:
is this the best name we can think of? What the user has to know anout this to register a payment?

Reply:
ephemeral highlights the fact that the user should delete the key after signing the tree. Open to suggestions